docs/source/quark_script.rst

```diff
@@ -1832,24 +1838,25 @@ Finally, we use Quark API ``quarkResultInstance.findMethodInCaller(callerMethod,
     quarkResult = runQuarkAnalysis(SAMPLE_PATH, ruleInstance)
 
     for accessExternalDir in quarkResult.behaviorOccurList:
+
         filePath = accessExternalDir.secondAPI.getArguments()[2]
 
         if quarkResult.isHardcoded(filePath):
-	    continue
+            continue
 
         caller = accessExternalDir.methodCaller
         strMatchingAPIs = [
-	    api
-	    for api in STRING_MATCHING_API
-	    if quarkResult.findMethodInCaller(caller, api)
+                api for api in STRING_MATCHING_API if quarkResult.findMethodInCaller(
+                    caller, api)
         ]
 
         if not strMatchingAPIs:
-	    print(f"CWE-22 is detected in method, {caller.fullName}")
-
+            print(f"CWE-22 is detected in method, {caller.fullName}")
```

Comment:
Sphinx complains that there is no blank line between lines 1854 and 1855. Here is the message:

```
/workspaces/quark-engine/docs/source/quark_script.rst:1855: WARNING: Explicit markup ends without a blank line; unexpected unindent. [docutils]
```

Please add a blank line between lines 1854 and 1855.